pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -273,6 +273,21 @@ def test_to_json(self):
         expected = OrderedDict([('schema', schema), ('data', data)])
         assert result == expected
 
+    def test_to_json_orient(self):
+        s1 = pd.Series([1, 2, 3], name='a')
+        df1 = pd.DataFrame(s1)
+
+        result_series = s1.to_json(orient='records')
+        result_df = df1.to_json(orient='records')
+        assert result_df == result_series
```

Comment:
Instead of comparing to a DataFrame can you manually construct the expected JSON? Would be a more explicit test

Reply:
changed!